### ISSUE TITLE
Uncaught error when accidentally specifying incorrect primer plates

### ISSUE THREAD

**Issue by wasade** [jdereus/labman#471]
If a user selects an i7 plate for "Select i5 primer plate" and a i5 plate for "Select i7 primer plate", an exception occurs.

Traceback is as follows

```bash
AN ERROR HAS OCCURED!
Please copy the following into an issue in our github repository

Error
labman.shotgun_primer_set with ID 'None' does not exist

Traceback
Traceback (most recent call last): 
File "/Users/dtmcdonald/miniconda3/envs/labcontrol/lib/python3.7/site-packages/tornado/web.py", line 1697, in _execute result = method(*self.path_args, **self.path_kwargs) 
File "/Users/dtmcdonald/miniconda3/envs/labcontrol/lib/python3.7/site-packages/tornado/web.py", line 3174, in wrapper return method(self, *args, **kwargs) 
File "/Users/dtmcdonald/ResearchWork/software/labman/labcontrol/gui/handlers/process_handlers/library_prep_shotgun_process.py", line 68, in post for pid, plate_name, i5p, i7p in json_decode(plates_info)] 
File "/Users/dtmcdonald/ResearchWork/software/labman/labcontrol/gui/handlers/process_handlers/library_prep_shotgun_process.py", line 68, in for pid, plate_name, i5p, i7p in json_decode(plates_info)] 
File "/Users/dtmcdonald/ResearchWork/software/labman/labcontrol/db/process.py", line 1472, in create TRN.execute_fetchlast()) 
File "/Users/dtmcdonald/ResearchWork/software/labman/labcontrol/db/base.py", line 40, in __init__ raise exceptions.LabControlUnknownIdError(self._table, id_) 
labcontrol.db.exceptions.LabControlUnknownIdError: labman.shotgun_primer_set with ID 'None' does not exist 
```

![shotgun_library_index_juggling](https://user-images.githubusercontent.com/474290/56539470-e1998300-651a-11e9-83d1-b33e138b5a86.gif)

**Comment by AmandaBirmingham** [comment]
So ... this suggests to me that https://github.com/jdereus/labman/issues/372 , "Limit i5/i7 primer plate selection in shotgun library prep to relevant working primer plates" was closed prematurely, since it appears that users still have options to select working plates that don't meet the specifications of the dropdown label and the back-end handler.  Will reopen that issue and merge with this one.

**Comment by AmandaBirmingham** [comment]
Closed as a knock-on effect of #372
